ui/pages/confirmations/components/security-provider-banner-alert/blockaid-banner-alert/blockaid-banner-alert.js

```diff
@@ -77,6 +110,35 @@ function BlockaidBannerAlert({ txData, ...props }) {
   } = securityAlertResponse;
 
   if (resultType === BlockaidResultType.Benign) {
+    if (isLoadingDisplayed) {
+      return (
+        <Box
+          alignItems={AlignItems.center}
+          borderColor={BorderColor.borderDefault}
+          borderRadius={BorderRadius.SM}
+          padding={2}
+          display={Display.Flex}
+        >
+          <Box
+            alignItems={AlignItems.center}
+            backgroundColor={TextColor.textDefault}
+            borderRadius={BorderRadius.full}
+            className="blockaid_banner__icon_wrapper"
+            display={Display.Flex}
+            justifyContent={JustifyContent.center}
+          >
+            <Icon
+              name={IconName.Check}
+              color={IconColor.primaryInverse}
+              size={IconSize.Sm}
+            />
+          </Box>
+          <Text color={TextColor.textDefault} marginLeft={2}>
+            {t('blockaidNoAlerts')}
+          </Text>
+        </Box>
+      );
+    }
```

Comment:
why wrap the 'blockaidNoAlerts' UI in `isLoadingDisplayed` if we display it for all Benign transactions?